src/NUnitEngine/Addins/nunit.v2.driver.tests/nunit.v2.driver.tests.csproj

```diff
@@ -30,6 +30,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="nunit.core.interfaces">
+      <HintPath>..\nunit.v2.driver\lib\nunit.core.interfaces.dll</HintPath>
+    </Reference>
```

Comment:
Another outcome of adding the unit test here: NUnit V2 tests are not supposed to reference the nunit.core.interfaces - the only exception, of course, being NUnit's own tests. Like the reference to the driver, this reference also makes the overall test less realistic.